src/autopas/molecularDynamics/MoleculeLJ.h

```diff
@@ -51,17 +51,24 @@ class MoleculeLJ final : public Particle {
                                        floatType /*z*/, floatType /*fx*/, floatType /*fy*/, floatType /*fz*/,
                                        size_t /*typeid*/, OwnershipState /*ownershipState*/>::Type;
 
+  /**
+   * Non-const getter for the pointer of this object.
+   * @tparam attribute Attribute name.
+   * @return this.
+   */
+  template <AttributeNames attribute, std::enable_if_t<attribute == AttributeNames::ptr, bool> = true>
+  constexpr typename std::tuple_element<attribute, SoAArraysType>::type::value_type get() {
+    return this;
+  }
```

Comment:
Potentially, one could also add a method that returns a const ptr, don't know whether that could be needed anywhere...

Reply:
sure but i would only add this if needed